### PR TITLE
Feature/add google analytics

### PR DESCRIPTION
**Pull request by adrianq**
### :pushpin: References

-   **Issue:** Closes https://app.clickup.com/t/22k4xqp

### :memo: Implementation

Added Google Analytics (Universal Analytics) using [react-ga](https://github.com/react-ga/react-ga). Added events and pageviews in every page.

### :fire: Testing

Analytics page: https://analytics.google.com/analytics/web/#/realtime/
Surf between pages and see in analytics page under realtime view. You can also try to search something and it will fire an event; you will be able to view the debug log in the console.

### CODE REVIEW

**Comment by p3rcypj** [comment]
The duplication of page view was because the goTo hook. It was called two times with the same path. This was causing also a warning from [history ](https://github.com/remix-run/history). I've solve it cheking if the new path is not the same as the actual one.

**Comment by p3rcypj** [comment]
Anyways now sends 2 page views but: one for "/viewer/" and the other one for "/viewer/6zow+EMD-21375". I think this should be expected, so we can track when people really wants to view 6zow or they just came from /

File: app/assets/javascripts/webserver/webserver_init.js [hunk 1]
```diff
@@ -1,30 +1,81 @@
...
+    });
+    $j("#cta-example").bind("click", () => {
+        gtag("event", "view_item", {
+            event_category: "call_to_action",
+            event_label: $j("#cta-example").data("id"),
+        });
+    });
 
+    const form = document.getElementById("form-search");
+    form.addEventListener("submit", function (event) {
```
[the remark this answers is not in the document]
Comment (p3rcypj) [reply app/assets/javascripts/webserver/webserver_init.js:46]: With jquery event listener style doesn't work. Cause when the submit is triggered, it executes the declared function. So it does not submit cause it was prevented. Anyways, I've changed the selector to class and querySelector; that should work.

[the remark this answers is not in the document]
Comment (tokland) [reply app/assets/javascripts/webserver/webserver_init.js:46]: Using id or class, document.querySelector still may return `null`, if a particular page does not have the element, right? I think we need a check.

[the remark this answers is not in the document]
Comment (p3rcypj) [reply app/assets/javascripts/webserver/webserver_init.js]: Yes, my bad. I've added that check

**Review by tokland — APPROVED** [review]
All good! :+1:

**Pull request merged**